### ISSUE TITLE
Broken machines list

### ISSUE THREAD

**Issue by steverydz** [canonical-web-and-design/maas-ui#488]
When loading `/r/machines` the page only shows an error:

```
TypeError: Cannot add property normalisedStatus, object is not extensible
```
This appears to have been introduced by https://github.com/canonical-web-and-design/maas-ui/pull/486 but at some point was working.

**Comment by Caleb-Ellis** [comment]
@steverydz Taking a look at the original PR, I think the `normalisedStatus` part should actually be a new selector, i.e. `machineSelectors.normalised`.

**Comment by steverydz** [comment]
Fixed by #492
